src/MusicStore/Controllers/HomeController.cs

```diff
@@ -60,9 +60,8 @@ public IActionResult AccessDenied()
             // Group the order details by album and return
             // the albums with the highest count
 
-            // TODO [EF] We don't query related data as yet, so the OrderByDescending isn't doing anything
             return await dbContext.Albums
-                .OrderByDescending(a => a.OrderDetails.Count())
+                .OrderByDescending(a => a.OrderDetails.Count)
```

Comment:
We only need to include them back if we want to materialize the Albums.OrderDetails collection. Since we don't need them in Views/Home/Index.cshtml (and because of https://github.com/aspnet/EntityFramework/issues/4695), we are ok leaving them out for now.